lib/packages/core/src/address/address.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Original work Copyright (c) 2021 Burst Apps Team
+ */
+import {getAccountIdFromPublicKey} from '@burstjs/crypto';
```

Comment:
I'm not happy to have this VO in the core package as BurstValue, and BurstTime are in `util`. But the public Key calcs rely on `crypto` and I don`t wanna make util dependent on `crypto`. But probably, this is inevitable? :shrug: 

Wdyt? @blankey1337 Should crypto be a dependency on util? so we'd have crypto --> util --> core?